store-implementation/federated-store/src/main/java/uk/gov/gchq/gaffer/federatedstore/util/MergeElementFunction.java

```diff
@@ -152,4 +220,12 @@ public Iterable<Object> apply(final Object update, final Iterable<Object> state)
             throw new GafferRuntimeException("Error getting all elements from temporary graph, due to:" + e.getMessage(), e);
         }
     }
+
+    private static Graph getGraph(final Map<String, Object> context) {
```

Comment:
Do we need static on this method considering it is private?

Reply:
This is only used by this class so it is private.
It is not tied to the instance of the class, it a utility method for fetching graphs out of context which has been configured by this class. so it can be static.

Is there a reason for this to not be private static?

Reply:
https://github.com/gchq/Gaffer/pull/3165